src/Microsoft.OpenApi/Microsoft.OpenApi.csproj

```diff
@@ -37,7 +37,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Dependabot is just going to create another PR to fix this.  We should keep it at 1.1.1